pubspec.yaml

```diff
@@ -1,19 +1,19 @@
 name: yaml
-version: 2.3.0-dev
+version: 2.3.0-nullsafety.0
```

Comment:
We should be doing a breaking change here, `3.0.0-nullsafety.0`

Reply:
Last published was ` 2.2.1`.  I thought `2.3.0` would suffice?

Reply:
`2.3.0` is a non-breaking feature release. We were using that versioning scheme for packages which are pinned by `package:flutter` (or other packages that ship in that SDK, using pub deps from an SDK package means those deps can't do major version bumps).

For package that _can_ do major version bumps, we are doing them. The recommendation here is `3.0.0-nullsafety.0`.

Reply:
OK, cool.  Thanks.